app/Http/Controllers/Webapp/SiteController.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: macintosh
+ * Date: 6/8/16
+ * Time: 11:40 AM
+ */
+
+namespace App\Http\Controllers\Webapp;
+
+
+use App\Http\Classes\RouteMap;
+use App\Libs\CdsApiDriver;
+use App\Libs\MessagesContainer\Error;
+use App\Libs\MessagesContainer\ErrorSet;
+use App\Libs\MessagesContainer\Message;
+use App\Libs\MessagesContainer\MessageSet;
+use App\ModelFactory\SiteFactory;
+use App\Models\Category;
+use App\Models\Post;
+use App\Models\Site;
+use App\Models\User;
+use Illuminate\Http\Request;
+
+class SiteController extends NeedAuthController
+{
+    public function __construct() {
+        parent::__construct();
+        if($this->currentUser->is('admin') === false) {
+            exit('404 Not Found');
+        }
+    }
+
+    public function getIndex() {
+        $sites = Site::all();
+
+        return view('webapp/site/index', [
+            'sites' => $sites
+        ]);
+    }
+
+    public function getCreate() {
+        return view('webapp/site/create');
+    }
+
+    public function postCreate(Request $request) {
+        $siteName = $request->get('site_name');
+        $siteApi = $request->get('site_api');
+        $siteUrl = $request->get('site_url');
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "site_url" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.relipa.blog/coding_rules#rule_key=php%3AS1192)